src/signals/settings/departments/Detail/components/CategoryLists/index.js

```diff
@@ -4,11 +4,7 @@ import styled from 'styled-components';
 import { Row, themeSpacing } from '@datapunt/asc-ui';
 import isEqual from 'lodash.isequal';
 
-import {
-  ControlsWrapper,
-  Fieldset,
-  Form,
-} from 'signals/incident-management/components/FilterForm/styled';
+import { ControlsWrapper, Fieldset, Form } from 'signals/incident-management/components/FilterForm/styled';
```

Comment:
Is anything changed in this file? I can't figure out.

Reply:
No, I guess not. At first, I made changes to this file, but I reverted them. Only formatting seems to be left.